src/Symfony/Component/Serializer/Mapping/AttributeMetadata.php

```diff
@@ -114,6 +125,14 @@ public function setSerializedName(string $serializedName = null)
         $this->serializedName = $serializedName;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function addSerializedName(string $serializedName = null, array $groups = [])
```

Comment:
Why allowing `null` as name?

Reply:
gg, I was influenced by the previous code but indeed no reason at all !
done